pkg/logs/config/channel_message.go

```diff
@@ -27,13 +27,14 @@ type Lambda struct {
 }
 
 // NewChannelMessageFromLambda construts a message with content and with the given timestamp and Lambda metadata
-func NewChannelMessageFromLambda(content []byte, utcTime time.Time, ARN, reqID string) *ChannelMessage {
+func NewChannelMessageFromLambda(content []byte, utcTime time.Time, ARN, reqID string, isError bool) *ChannelMessage {
 	return &ChannelMessage{
 		Content:   content,
 		Timestamp: utcTime,
 		Lambda: &Lambda{
 			ARN:       ARN,
 			RequestID: reqID,
 		},
+		IsError: isError,
```

Comment:
It seems the new field is only used in the tests, can it be removed?

Reply:
The new field is being used in `pkg/serverless/logs/logs_collector.go` and has existed as part of the `ChannelMessage` type prior to this change. Even though it was added previously, it was unfortunately never used :/